flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/storage/BufferAccumulator.java

```diff
@@ -46,7 +46,10 @@ public interface BufferAccumulator extends AutoCloseable {
      * transformed into finished buffers.
      */
     void receive(
-            ByteBuffer record, TieredStorageSubpartitionId subpartitionId, Buffer.DataType dataType)
+            ByteBuffer record,
+            TieredStorageSubpartitionId subpartitionId,
+            Buffer.DataType dataType,
+            boolean isBroadcast)
```

Comment:
For a broadcast-only partition, When isBroadcast is true, the subpartitionId is 0. 

For a non-broadcast-only partition (for exampe, a all-to-all edge), this will be all the subpartitionId range from 0 to numSupartitions(This is from a for loop).